wesnoth-l.po

```diff
@@ -23,6 +23,32 @@ msgstr ""
 msgid "Liberty"
 msgstr "Svoboda"
 
+# Alexander Slávik <slavik.alexander@seznam.cz>, 2007.
+# Vladimír Slávik <slavik.vladimir@seznam.cz>, 2007 - 2011.
+# Honza <Honza-mistic@seznam.cz>, 2014.
+# Martin Hrubý <hrubymar10@gmail.com>, 2017 - 2020.
+# Michal Žejdl <lachim@emer.cz>, 2019 - 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Battle for Wesnoth\n"
+"Report-Msgid-Bugs-To: https://bugs.wesnoth.org/\n"
+"POT-Creation-Date: 2022-01-02 18:01 UTC\n"
+"PO-Revision-Date: 2022-01-05 20:22+0100\n"
+"Last-Translator: Septim <septim@centrum.cz>\n"
+"Language-Team: Czech https://wiki.wesnoth.org/CzechTranslation\n"
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#. [campaign]: id=Liberty
+#: data/campaigns/Liberty/_main.cfg:10 data/campaigns/Liberty/_main.cfg:11
+msgid "Liberty"
+msgstr "Svoboda"
+
```

Comment:
Povedlo se vám tam vlepit znova kousek začátku. Nedaří se mi navrhnout (přes suggestion) správnou opravu. Opravíte to?

Reply:
Mělo by to být.
Vůbec mi tato změna hodně zlobila, doufám, že jsem snad došel k vylepšení a ne ke zhoršení...

Reply:
Je to v pořádku, díky.